virtcontainers/api.go

```diff
@@ -653,3 +653,23 @@ func StatsContainer(sandboxID, containerID string) (ContainerStats, error) {
 
 	return s.StatsContainer(containerID)
 }
+
+// AddNetwork is the virtcontainers entry point to add a new nic.
+func AddNetwork(sandboxID, ifName string) error {
+	if sandboxID == "" {
+		return errNeedSandboxID
+	}
+
+	lockFile, err := rLockSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+	defer unlockSandbox(lockFile)
+
+	s, err := fetchSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+
+	return s.AddNetwork(ifName)
+}
```

Comment:
I guess you will complete this by providing the implementation for `UpdateRoutes()`, `ListNetwork()`, and `UpdateNetwork()` ?